gnomad_qc/v4/create_release/validate_and_export_vcf.py

```diff
@@ -835,23 +859,31 @@ def populate_info_dict(
         combinations.
     :param data_type: Data type to populate info dict for. One of "exomes" or
         "genomes". Default is "exomes".
+    :param freq_comparison_included: Whether frequency comparison data is included in the HT.
+    :param extra_suffix: Suffix to add to INFO field.
+    :param extra_description_text: Extra description text to add to INFO field.
     :return: Updated INFO dictionary for VCF export.
     """
-    # Get existing info fields from predefined info_dict, e.g. `FS`,
-    # `non_par`, `negative_train_site`...
-    vcf_info_dict = info_dict.copy()
-    vcf_info_dict = {f: vcf_info_dict[f] for f in info_fields if f in vcf_info_dict}
-
-    # Add allele-specific fields to info dict, including AS_VQSR_FIELDS
-    vcf_info_dict.update(
-        add_as_info_dict(info_dict=info_dict, as_fields=AS_FIELDS + AS_VQSR_FIELDS)
-    )
+    vcf_info_dict = {}
+    if data_type == "joint":
+        if "joint" in subset_list:
+            vcf_info_dict.update(JOINT_REGION_FLAGS_INFO_DICT)
+    else:
+        # Get existing info fields from predefined info_dict, e.g. `FS`,
+        # `non_par`, `negative_train_site`...
+        vcf_info_dict.update(info_dict)
+        vcf_info_dict = {f: vcf_info_dict[f] for f in info_fields if f in vcf_info_dict}
+        # Add allele-specific fields to info dict, including AS_VQSR_FIELDS
+        vcf_info_dict.update(
+            add_as_info_dict(info_dict=info_dict, as_fields=AS_FIELDS + AS_VQSR_FIELDS)
+        )
 
     for subset in subset_list:
         subset_pops = deepcopy(pops)
-        if (subset == "joint") | (data_type == "genomes"):
-            subset_pops.update({"ami": "Amish"})
-        description_text = "" if subset == "" else f" in {subset} subset"
+        if data_type == "joint":
+            description_text = f" in {subset} dataset " if subset != "" else ""
```

Comment:
```suggestion
            description_text = f" in {subset} dataset" if subset != "" else ""
```
This either always ends a sentence or the following word has a space prefixed so lets drop the extra space.